include/pros/rotation.hpp

```diff
@@ -0,0 +1,149 @@
+/**
+ * \file pros/rotation.hpp
+ *
+ * Contains prototypes for functions related to the VEX Rotation sensor.
+ *
+ * Visit https://pros.cs.purdue.edu/v5/tutorials/topical/rotation.html to learn
+ * more.
+ *
+ * This file should not be modified by users, since it gets replaced whenever
+ * a kernel upgrade occurs.
+ *
+ * Copyright (c) 2017-2020, Purdue University ACM SIGBots.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+#ifndef _PROS_ROTATION_HPP_
+#define _PROS_ROTATION_HPP_
+
+#include <cstdint>
+
+#include "pros/rotation.h"
+
+namespace pros {
+class Rotation {
+	const std::uint8_t _port;
+
+	public:
+	Rotation(const std::uint8_t port) : _port(port){};
+
+	/**
+	 * Resets Rotational Sensor
+	 *
+	 * Resets rotation presumably to 0.
```

Comment:
```suggestion
	 * Resets rotation to 0.
```
? (Also shouldn't be different from the c comment)